knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -154,6 +156,31 @@ func (r *ReconcileKnativeServing) ensureCustomCertsConfigMap(instance *servingv1
 	return nil
 }
 
+// Install Kourier Ingress Gateway
+func (a *ReconcileKnativeServing) installKourier(instance *servingv1alpha1.KnativeServing) error {
+	log.Info("Installing Kourier Ingress")
+	const path = "deploy/resources/kourier/kourier.yaml"
+
+	manifest, err := mf.NewManifest(path, false, a.client)
+	if err != nil {
+		log.Error(err, "Unable to create Kourier Ingress install manifest")
+		return err
+	}
+	transforms := []mf.Transformer{mf.InjectOwner(instance)}
+	// let's hardcode this for now.
+	transforms = append(transforms, mf.InjectNamespace("knative-serving-ingress"))
+
```

Comment:
Can we filter the kourier-system namespace out here to avoid having to manipulate a release manifst?

Reply:
Yes, we can. And I realized that `namespace` is transformed to create namespace by injectNamespace, so we just don't need to comment it out.

https://github.com/openshift-knative/serverless-operator/blob/55d9f27c68286fb9be22fc03f3c3aa842db87054/knative-operator/vendor/github.com/jcrossley3/manifestival/transform.go#L42-L43